### ISSUE TITLE
Resilver restarts everytime zpool status command is executed

### ISSUE THREAD

**Issue by eggehad** [zfsonlinux/zfs#6432]
Operational ZFS running v0.6.5.10 with 2x 18 disk VDEVs, in which I had previously completed 14 resilvers with zero issues, was upgraded this afternoon to v0.7.0
After upgrade and restart, a new resilver was started, and an hour later, I checked on the status by doing a zpool status command.  Command showed the resilver had just started.  Odd..
Waited 2 hours, did it again, again, showed the resilver had just started.  Each of the last 5 times, the resilver appears to restart when I execute the zpool status command.  
very oddly I get the same completion percentage each time I run it, and it's now been running for more than 5 hours, normally it's at 20+% at this point.

 scan: resilver in progress since Sun Jul 30 16:43:20 2017
	396G scanned out of 110T at 2.85G/s, 10h57m to go
	229M resilvered, 0.35% done 
 scan: resilver in progress since Sun Jul 30 18:35:56 2017
	397G scanned out of 110T at 3.89G/s, 8h2m to go
	296M resilvered, 0.35% done
 scan: resilver in progress since Sun Jul 30 18:38:41 2017
	394G scanned out of 110T at 1.91G/s, 16h24m to go
	148M resilvered, 0.35% done  
scan: resilver in progress since Sun Jul 30 18:47:09 2017
	395G scanned out of 110T at 3.77G/s, 8h18m to go
	198M resilvered, 0.35% done
 scan: resilver in progress since Sun Jul 30 18:53:30 2017
	395G scanned out of 110T at 1.92G/s, 16h15m to go
	164M resilvered, 0.35% done

currently running on:
Fedora Core 25, Kernel: 4.11.12-200.fc25.x86_64
ZFS 0.7.0-1
SPL 0.7.0-1

**Comment by eggehad** [comment]
I downgraded zfs-0.7.0-1 to 0.6.5.11, and when running zpool status, after the usual status header, it followed with "Aborted,"  with no drive or vdev info, which was very concerning.
So I uninstalled zfs, dkms, spl, nvidia, ... completely, and did a complete re-install to 0.7.0-1.  

Problem stated above is now mysteriously gone.  Running zpool status now gives the expected results:
**************
pool: media-pool
state: ONLINE
status: One or more devices is currently being resilvered.  The pool will
        continue to function, possibly in a degraded state.
action: Wait for the resilver to complete.
  scan: resilver in progress since Sun Jul 30 19:19:55 2017
        25.9T scanned out of 110T at 2.61G/s, 9h13m to go
        20.7G resilvered, 23.47% done
config:
**************************
Not sure where the true source of problem lay, so this thread can probably just be closed, or turned into an installation bug.  downgrading, then uninstalling then re-installing seems to have solved the issue.

**Comment by behlendorf** [comment]
@eggehad thanks for following up.  I'm glad things are working properly as a clean install, but I agree it's troubling.  Perhaps you were running with a mismatched user and kernel packages?  Closing.

**Comment by eggehad** [comment]
Turns out this *IS* a problem.  After that drive finished resilvering, which by the way, was the longest resilver time I have ever had, I started a new one.  This new resilver has exactly the same problem, everytime I run zpool status, a long pause ensues, and the resilvering process restarts.   Also notice the magic 0.35% completion, which shows up even though just a minute earlier the percentage was 0% 

Any ideas how I can help to debug this problem?
***
[david@pas plexpy]$ sudo zpool status
  pool: a-media-pool
 state: ONLINE
status: One or more devices is currently being resilvered.  The pool will
        continue to function, possibly in a degraded state.
action: Wait for the resilver to complete.
  scan: resilver in progress since Tue Aug  1 20:20:20 2017
        14.3M scanned out of 110T at 14.3M/s, (scan is slow, no estimated time)
        0B resilvered, 0.00% done
config:


***
[david@pas plexpy]$ sudo zpool status
  pool: a-media-pool
 state: ONLINE
status: One or more devices is currently being resilvered.  The pool will
        continue to function, possibly in a degraded state.
action: Wait for the resilver to complete.
  scan: resilver in progress since Tue Aug  1 20:21:18 2017
        14.9M scanned out of 110T at 14.9M/s, (scan is slow, no estimated time)
        412K resilvered, 0.00% done
config:


***
[david@pas plexpy]$ sudo zpool status
  pool: a-media-pool
 state: ONLINE
status: One or more devices is currently being resilvered.  The pool will
        continue to function, possibly in a degraded state.
action: Wait for the resilver to complete.
  scan: resilver in progress since Tue Aug  1 20:22:40 2017
        395G scanned out of 110T at 1.92G/s, 16h19m to go
        157M resilvered, 0.35% done


***
[david@pas plexpy]$ sudo zpool status
  pool: a-media-pool
 state: ONLINE
status: One or more devices is currently being resilvered.  The pool will
        continue to function, possibly in a degraded state.
action: Wait for the resilver to complete.
  scan: resilver in progress since Tue Aug  1 20:25:24 2017
        14.6M scanned out of 110T at 14.6M/s, (scan is slow, no estimated time)
        4K resilvered, 0.00% done
config:
